src/test/java/org/minions/devfund/noemiguzman/moviedatabase/ActorTest.java

```diff
@@ -0,0 +1,57 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * class to test actor.
+ */
+public class ActorTest {
+    private Actor actor1;
+    private Movie movie2;
+    private ArrayList<Movie> movieList1;
+
+    /**
+     * Set movie 1 and movie2.
+     */
+    @Before
+    public void setUpActors() {
+        Movie movie1 = new Movie("Batman");
+        movie2 = new Movie("Batman and Robin");
+
+        movieList1 = new ArrayList<>();
+        movieList1.add(movie1);
+
+        actor1 = new Actor("Christian Bale", movieList1);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Christian Bale" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)